web/ui/react-app/src/utils/func.test.ts

```diff
@@ -0,0 +1,7 @@
+import { uuidGen } from './func';
```

Comment:
I believe the whole `func.*` files can be removed since `uuidGen` isn't used anywhere anymore.